src/wallet/PegOut.cpp

```diff
@@ -19,7 +20,7 @@ mw::Transaction::CPtr PegOut::CreatePegOutTx(
     std::vector<Input> inputs = WalletUtil::SignInputs(input_coins);
 
     // Calculate fee
-    const uint64_t fee = WalletUtil::CalculateFee(fee_base, input_coins.size(), 1, 2);
+    const uint64_t fee = fee_base * Weight::Calculate({ .num_kernels = 1, .num_owner_sigs = 1, .num_outputs = 2 });
```

Comment:
Technically there is only one output, not sure if that's significant.

Reply:
Yea, this is also paying for the canonical output that the miner creates. Technically it should have its own fee calculation logic too, but I didn't want to over complicate it